cpmpy/expressions/python_builtins.py

```diff
@@ -34,9 +36,9 @@ def all(iterable):
     if isinstance(iterable, NDVarArray): iterable=iterable.flat # 1D iterator
     collect = [] # logical expressions
     for elem in iterable:
-        if elem is False or elem is np.False_:
-            return False  # no need to create constraint
-        elif elem is True or elem is np.True_:
+        if is_false_cst(elem):
+            return BoolVal(False)  # no need to create constraint
```

Comment:
I don't think we can/should in general change the return type of a Python builtin...

'any' can be used a lot in an arbitrary code-base that only includes cpmpy for a small part

maybe if we check that at least one element is a CPMpy expr, but for now I will just undo this